packages/react-native-client/src/components/FishjamRoom/parsePeersToTracks.ts

```diff
@@ -0,0 +1,47 @@
+import { PeerWithTracks } from '../../hooks/usePeers';
+import { GridTrack } from './GridTrackItem';
+
+const createGridTracksFromPeer = (peer: PeerWithTracks): GridTrack[] => {
+  const tracks: GridTrack[] = [];
+
+  if (peer.cameraTrack && peer.cameraTrack.isActive) {
+    tracks.push({
+      track: peer.cameraTrack,
+      peerId: peer.id,
+      isLocal: peer.isLocal,
+      isVadActive: peer.microphoneTrack?.vadStatus === 'speech',
+    });
+  }
+
+  if (peer.screenShareVideoTrack && peer.screenShareVideoTrack.isActive) {
+    tracks.push({
+      track: peer.screenShareVideoTrack,
+      peerId: peer.id,
+      isLocal: peer.isLocal,
+      isVadActive: peer.screenShareAudioTrack?.vadStatus === 'speech',
+    });
+  }
+
+  if (tracks.length === 0) {
+    tracks.push({
+      track: null,
+      peerId: peer.id,
+      isLocal: peer.isLocal,
+      isVadActive:
+        peer.microphoneTrack?.vadStatus === 'speech' ||
```

Comment:
That's a lot of `speech` in this file. Maybe we could extract it to some variable instead of comparing directly to string?

Reply:
This comes from `VadStatus` type so it's typed checked. 
`type VadStatus = 'silence' | 'speech';`